.bazelversion

```diff
@@ -0,0 +1 @@
+1.2.0
```

Comment:
Could you add a comment here to keep in sync with build_defs/bazel_version_check.bzl ?

Also modify that file to say to keep in sync with this one.

Reply:
Good idea, done